base/sparse/cholmod.jl

```diff
@@ -576,7 +617,7 @@ for Ti in IndexTypes
             d
         end
 
-        function spsolve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
+        function solve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
```

Comment:
The idea is to match the names in CHOLMOD for the wrappers. Then CHOLMOD can be the documentation for wrappers as well. If we begin to change that it will require documenting all the wrapper function ourself which I think would be the a waste of time.